### PR TITLE
Added HTTP chunked flush

### PR DESCRIPTION
**Pull request by lsviben-codi**
Temporary fix for our watch problem. 

Should be refactored to something cleaner and configurable and sent upstream.

### CODE REVIEW

File: httpproxy.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ import (
 
 	"github.com/mmatczuk/go-http-tunnel/log"
 	"github.com/mmatczuk/go-http-tunnel/proto"
+	"time"
```
Comment (kwapik) [inline httpproxy.go]: Move one block up.

File: utils.go [hunk 2]
```diff
@@ -11,9 +11,24 @@ import (
 	"strings"
 
 	"github.com/mmatczuk/go-http-tunnel/log"
+	"time"
+	"sync"
```
Comment (kwapik) [inline utils.go]: ditto

File: httpproxy.go [hunk 3]
```diff
@@ -62,6 +63,7 @@ func NewMultiHTTPProxy(localURLMap map[string]*url.URL, logger log.Logger) *HTTP
 		logger:      logger,
 	}
 	p.ReverseProxy.Director = p.Director
+	p.FlushInterval = time.Millisecond * 500
```
Comment (kwapik) [inline httpproxy.go]: It's not used anywhere.

Comment (lsviben-codi) [reply httpproxy.go]: Its used in the ReverseProxy

Comment (lsviben-codi) [reply httpproxy.go]: Ill change it to `p.ReverseProxy.FlushInterval = time.Millisecond * 500` to make it more clear.

**Pull request merged**